cryptoorg/chain.json

```diff
@@ -1,24 +1,142 @@
 {
-    "$schema": "../chain.schema.json",
-    "chain_name": "cryptoorg",
-    "status": "live",
-    "network_type": "mainnet",
-    "pretty_name": "Crypto.org",
-    "chain_id": "crypto-org-chain-mainnet-1",
-    "bech32_prefix": "cro",
-    "slip44": 394,
-    "apis": {
-        "rpc": [
-            {
-                "address": "https://rpc-crypto-org.keplr.app/",
-                "provider": "chainapsis"
-            }
-        ],
-        "rest": [
-            {
-                "address": "https://lcd-crypto-org.keplr.app/",
-                "provider": "chainapsis"
-            }
-        ]
+  "$schema": "../chain.schema.json",
+  "chain_name": "crypto-org-chain",
```

Comment:
I was trying to avoid special characters in the chain names.  To avoid debate over whether it should be things like "cosmoshub" vs "cosmos-hub".

Do you think `cryptoorgchain` is okay?  Or is that silly?

Reply:
Special characters seem fine. Should match the official nomenclature the chain itself uses, spaces are fine as well imo.

Reply:
I'm gonna go ahead and make this `cryptoorgchain` for now.  We can revisit in the future.

Reply:
```suggestion
  "chain_name": "cryptoorgchain",
```